.github/workflows/docs_build_and_deploy.yml

```diff
@@ -27,7 +27,9 @@ jobs:
     needs: linting
     runs-on: ubuntu-latest
     steps:
-      - uses: neuroinformatics-unit/actions/build_sphinx_docs@v2
+      - uses: neuroinformatics-unit/actions/build_sphinx_docs@main
```

Comment:
Is there a reason to use `main` here? I think in general we want to be using the latest released version (in case a rogue PR goes through). `@v2` will resolve to the latest `v2.x` release.

Reply:
I think this is a regression, or I completely lost my head and did it on purpose at some point in the past. I have no idea why it's like that, have reverted!